src/features/create-dao/CreateDAOForm.tsx

```diff
@@ -0,0 +1,42 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Style Imports
+import styles from './CreateDAOForm.module.scss';
+
+//- Component Imports
+import { Wizard } from '@zero-tech/zui/components/Wizard/Wizard';
+import { CreateDAOBody } from './CreateDAOBody';
```

Comment:
nit: Could add another `index.ts` file under `create-dao` and export all and update these imports to one line